src/iotile/cloud.service.ts

```diff
@@ -730,9 +730,21 @@ export class CloudService {
 
   public getDataBlock(dataBlockSlug: string): Observable<DataBlock> {
     let url = '/datablock/' + dataBlockSlug + '/';
-
+    
     return this._get(url).map((data: any) => {
       return new DataBlock(data);
     }, err => console.error(err));
   }
+
+  public postDataBlock(
```

Comment:
Please follow the way other post/patch functions work. The argument should be a `DataBlock` and the DataBlock should have a serialization function to create the payload.

Please see `postOrg` as an example.